base_location_nuts/wizard/nuts_import.py

```diff
@@ -190,7 +190,8 @@ def run_import(self):
         # All current NUTS (for available countries),
         #   delete if not found above
         nuts_to_delete = nuts_model.search(
-            [('country_id', 'in', [x.id for x in self._countries.values()])])
+            [('country_id', 'in',
+              [x.id for x in list(self._countries.values())])])
```

Comment:
No need for `list()` here. Views are better when iterating.

Reply:
The same about the automatism of `2to3`